include/bio/record.hpp

```diff
@@ -67,15 +67,15 @@ enum class field : uint64_t
     /*_private*/
 
     // Fields unique to variant io
-    chrom = ref_id, //
+    chrom = ref_id, //!< CHROM field in Var I/O.
     /*pos*/
     /*id*/
-    ref   = optionals + 1,
-    alt,
+    ref   = optionals + 1, //!< REF field in Var I/O.
+    alt,                   //!< ALT field in Var I/O.
```

Comment:
Remember I wanted to ask something in our last meeting? That was it:

why do you need optionals + 1 here? But I checked in Godbolt. This is not intuitive. I thought the compiler can keep track. 

I think we need a test that checks if no number appears twice because when adding something you forget to update some +1 below or need to change the anchor.

Reply:
It stops auto-numbering as soon as you do one "=" inside 😑 Very annoying.
I am not sure we can test against this, because for some things, we want the same ID, or not?

Reply:
Well the test is going to be very ungeneric I guess but if you add something and the test then obviously fails and needs adjustments it might make you think twice about this problem?

Reply:
Yeah... or we just make all the ID entries unique?

Reply:
NO, I really liked the idea that you can access the same information with different names depending on the context! E.g. `rec.id()` works although SAM reads `field::qname` (right?)

Reply:
In any case, it is something, we can easily change later on. Do you want to add to our list of things to discuss later?

Reply:
https://github.com/h-2/b.i.o./wiki/Discussion-points